### PR TITLE
Filter by repositories

### LINKED ISSUE #50: Filter by repository
Now that NSS is also supported by code-review bot, we need a repository filter on the frontend.

1. Export the repository information in Taskcluster index
2. Use that new information in the frontend

### PR DESCRIPTION
**Pull request by La0**
Fixes #50

### CODE REVIEW

**Comment by La0** [comment]
Demo on testing : https://static-analysis.testing.moz.tools

To rebase after #52 is merged.

**Comment by La0** [comment]
Rebased on top of TC fix

File: frontend/src/Tasks.vue [hunk 1]
```diff
@@ -164,9 +158,9 @@ export default {
           </td>
 
           <td>
-            <span class="tag is-light" v-if="task.data.source == 'phabricator'">Phabricator</span>
-            <span class="tag is-primary" v-else-if="task.data.source == 'try'">Try</span>
-            <span class="tag is-danger" v-else>Unknown</span>
+            <span class="tag is-primary" v-if="task.data.repository == 'mozilla-central'">Mozilla Central</span>
```
Comment (marco-c) [inline frontend/src/Tasks.vue]: In the future, it might be worth populating this automatically somehow.

Comment (marco-c) [reply frontend/src/Tasks.vue]: (when we'll have more repos)

**Pull request merged**